apps/bfd-model/bfd-model-rif-samples/src/test/java/gov/cms/bfd/model/rif/samples/SampleDataColumnsTest.java

```diff
@@ -56,20 +58,17 @@ private void verifyColumns(StaticRifResourceGroup sampleGroup) {
       Enum<?>[] columnsInEnum = getColumnsInEnum(sampleFile.getRifFileType());
 
       // Use a CSVParser to parse the header out of the sample file.
-      CSVFormat parserFormat = CSVFormat.DEFAULT.withDelimiter('|');
+      CSVFormat parserFormat = CSVFormat.DEFAULT.builder().setDelimiter('|').build();
```

Comment:
All the changes to `.builder()` are to remove deprecation warnings on `withDelimiter`